docs/UserGuide.md

```diff
@@ -73,47 +73,62 @@ Shows a message explaning how to access the help page.
+### Adding a student: `add`
 
-Adds a person to the address book.
+Adds a student to the address book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A student can have any number of tags (including 0)
 </div>
 
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+### Listing all students : `list`
 
-Shows a list of all persons in the address book.
+Shows a list of all students in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a student : `edit`
 
-Edits an existing person in the address book.
+Edits an existing student in the address book.
 
 Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [ct/CLASS_TIMING] [r/HOURLY_RATE] [t/TAG]…​`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
+* When editing tags, the existing tags of the student will be removed i.e adding of tags is not cumulative.
+* You can remove all the student's tags by typing `t/` without
     specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-*  `edit 3 ct/Mon 1pm to 3pm r/$50` Edits the class timing and hourly rate of the 3rd person to be `Mon 1pm to 3pm` and `$50` respectively.
+*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st student to be `91234567` and `johndoe@example.com` respectively.
+*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd student to be `Betsy Crower` and clears all existing tags.
+*  `edit 3 ct/Mon 1pm to 3pm r/$50` Edits the class timing and hourly rate of the 3rd student to be `Mon 1pm to 3pm` and `$50` respectively.
 
-### Locating persons by name: `find`
+### Editing a student's nok : `edit`
 
-Finds persons whose names contain any of the given keywords.
+Edits an existing nok of a student in the address book (similar to above).
+
+Format: `edit INDEX nok/ [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] …​`
```

Comment:
Is it possible to combine it with the previous one? Because if we format it like this it may sound like u can either edit the NOK or edit the student's details, and not both at the same time.

Reply:
But other than this, LGTM!

Reply:
Ohh you want to edit both at the same time? But how do we do this

Reply:
Maybe you can format the `edit` command like this:

`edit INDEX fieldsOfStudentToEdit [nok/ fieldsOfNokToEdit]`? so everything after `nok/` is purely for `nok`.

For example `edit 1 n/kevin p/12345678 nok/ n/stuart p/ 87654321`. In this case we edit the name  and phone number of student 1 to become `kevin` and `12345678` and set his nok name to become `stuart` and number of nok to be `87654321`.

We can use the similar concept for add.

`add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​  [nok/ nokDetails]` where nokDetails consist of [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] which are all optional. @softmagnet you can refer to this when doing your portion of UserGuide.

This is just a suggestion.

Reply:
This sounds good, we can do it like this